build/pipelines/templates-v2/pipeline-onebranch-full-release-build.yml

```diff
@@ -78,6 +78,9 @@ extends:
     cloudvault: # https://aka.ms/obpipelines/cloudvault
       enabled: false
     globalSdl: # https://aka.ms/obpipelines/sdl
+      asyncSdl:
+        enabled: true
+        tsaOptionsFile: 'build/config/tsa.json'

```

Comment:
No `$(Build.SourcesDirectory)`?

Reply:
fear prevented me from doing that. The docs said “relative path to tsa options”, and I can’t tell if it’s an example or a requirement 😅 given OneBranch’s _whole deal_, I’d rather not even deviate from their docs